packages/gatsby-plugin-manifest/README.md

```diff
@@ -19,6 +19,11 @@ the created manifest.json.
 
 ## How to use
 
+There are three configurations in which this plugin will function: manual, hybrid, and auto. What config options you include will determine how the plugin functions.
+
+### Manual configuration
+In the manual configuration you are responsible for defining the entire web app manifest and providing the defined icons in the static directory. See the example below:
```

Comment:
It'd be really cool to see a 1-2 sentence use-case explained for each config option. Why would l choose manual vs. hybrid vs. automatic?